index.html

```diff
@@ -55,6 +77,169 @@
 
     -->
 
-    <h1>Assignment 01 - To Be Completed</h1>
+    <!--Main page content-->
+    <!--Header-->
+    <header>
+      <h1>Kirby Fan Club</h1>
+      <img
+        src="kirby.png"
+        alt="Illustrated graphic of Kirby character"/>
+      
+      <nav>
+        <h2>Site Menu</h2>
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#about">About Us</a></li>
+          <li><a href="#join">Join the Club</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <h2>Who is Kirby?</h2>
+
```

Comment:
Same as above, my recommendation here would be to have your `main` area have your `h1` and then your heading structure go down from there.